html/avatar.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/components/avatar.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>This is avatar</h1>
+   
+    <img class="avatarImg " src="/images/img_avatar.png" />
```

Comment:
add alt attributes for specific and alternate text for image

Reply:
okay thanks!